src/components/locator.ts

```diff
@@ -280,11 +285,7 @@ export class Locator {
             }
         }
 
-        let filePath = path.resolve( record.input.replace(/(\r\n|\n|\r)/gm, '') )
-        if (docker && process.platform === 'win32') {
-            filePath = path.resolve(path.dirname(pdfPath), record.input as string)
-        }
-
```

Comment:
After moving this block inside the  `else` part above, nothing deals with docker when using `synctexjs`.  This is a bit surprising. 

It looks like two things may be mixed up here and it works when using external `synctex`.  Have you tested that docker works both with external `synctex` and `synctexjs`.

Reply:
I do not have Windows so just to be sure have you tested that using docker and `synctexjs` works on Windows?

Reply:
I just tested this and `synctexjs` doesn't work with Docker on Windows anyway because it includes it's own check whether the file exists before we can patch the path: The block could be moved out of the else branch though.

Reply:
For the docker and `synctexjs` issue, it seems we just need to add something like
```js
if (docker && process.platform === 'win32') {
    record.input = record.input.replace('/data/', '')
}
```
juste before https://github.com/James-Yu/LaTeX-Workshop/blob/8dd06def151053ceecfec4bd90547863885a090b/src/components/synctex.ts#L206

Unfortunately I cannot test it. Could you make a PR or do you prefer me to do it and rely on you for the testing?

Reply:
I can make a PR. I also noticed that the cleaning command doesn't work if I build with Docker. I'll see whether I find further issues and will make a PR as soon as possible.